Packs/CommonPlaybooks/Playbooks/playbook-Enrichment_for_Verdict_README.md

```diff
@@ -1,51 +1,55 @@
-This playbook checks prior alert closing reasons and performs enrichment and prevalence checks on different IOC types. It then returns the information needed to establish the alert's verdict.
+This playbook checks prior alert closing reasons and performs enrichment and prevalence checks on different IOC types. It then  returns the information needed to establish the alert's verdict.
 
 ## Dependencies
+
 This playbook uses the following sub-playbooks, integrations, and scripts.
 
 ### Sub-playbooks
-* IP Enrichment - Generic v2
+
 * Domain Enrichment - Generic v2
 * URL Enrichment - Generic v2
-* Get prevalence for IOCs 
-* Account Enrichment - Generic v2.1
+* IP Enrichment - Generic v2
 * File Reputation
+* Get prevalence for IOCs
+* Account Enrichment - Generic v2.1
 
 ### Integrations
+
 This playbook does not use any integrations.
 
 ### Scripts
-* IsIntegrationAvailable
+
 * Set
 * SearchIncidentsV2
 
 ### Commands
-* wildfire-get-verdict
+
 * wildfire-report
+* wildfire-get-verdict
 
 ## Playbook Inputs
+
 ---
 
 | **Name** | **Description** | **Default Value** | **Required** |
 | --- | --- | --- | --- |
-| threshold | The number of previous alerts that were closed as false positive alerts. This threshold establishes whether the Previous Verdict key will be marked as false positive. | alert.hostip | Optional |
+| threshold | The number of previous alerts that were closed as false positive alerts. This threshold establishes whether the Previous Verdict key will be marked as false positive. | 2 | Optional |
```

Comment:
I assume that is just the format stuff, but note the default input was changed here ( when in the original file there was no change).